site/static/docs/5.3/assets/img/boosted-sprite.svg

```diff
@@ -1,56 +1,53 @@
   </symbol>
-  <symbol fill="currentColor" viewBox="0 0 30 30" id="buy">
-    <path d="M26.406 11.622H19.76V8.176c0-2.64-2.145-4.801-4.785-4.801s-4.785 2.16-4.785 4.801v3.446H3.35a1.5 1.5 0 00-1.286 2.272L7.85 26.625H21.8l5.892-12.731c.136-.225.258-.49.258-.772 0-.828-.715-1.5-1.544-1.5zm-21.556 3h5.25l.577 3.751H6.618zm4.5 9.753l-1.8-3.751h3.473l.577 3.75zm6.75 0h-2.4l-.6-3.751h3.6zm.9-6.002h-4.05l-.6-3.75h5.1zm.51-6.751H12.44l-.003-3.446a2.537 2.537 0 115.073 0zm2.94 12.753h-2.26l.61-3.751h3.45zm2.731-6.002H19.08l.471-3.75h5.255z"/>
+  <symbol fill="currentColor" viewBox="0 0 24 24" id="buy">
+    <path d="M21.205 9.31h-5.317V6.553c0-2.113-1.716-3.841-3.828-3.841S8.232 4.44 8.232 6.552V9.31H2.76a1.2 1.2 0 0 0-1.03 1.817l4.63 10.185h11.16l4.714-10.185c.108-.18.206-.391.206-.617 0-.663-.572-1.2-1.235-1.2ZM3.96 11.71h4.2l.462 3H5.375l-1.415-3Zm3.6 7.802L6.12 16.51h2.778l.462 3h-1.8Zm5.4 0h-1.92l-.48-3.001h2.88l-.48 3Zm.72-4.801h-3.24l-.48-3.001h4.08l-.36 3Zm.408-5.401h-4.055l-.003-2.757a2.03 2.03 0 1 1 4.059 0l-.001 2.757Zm2.352 10.202h-1.807l.487-3.001h2.76l-1.44 3Zm2.185-4.801h-3.282l.377-3.001h4.204l-1.299 3Z"/>
   </symbol>
-  <symbol fill="currentColor" viewBox="0 0 1000 1000" id="search">
-    <path d="M892.285,819.383L666.631,593.728a310.794,310.794,0,0,0,58.05-181.388c0-172.5-139.839-312.34-312.341-312.34S100,239.838,100,412.34,239.838,724.681,412.34,724.681a310.918,310.918,0,0,0,181.4-58.05L819.383,892.275a25.053,25.053,0,0,0,35.342,0l37.56-37.55A25.071,25.071,0,0,0,892.285,819.383ZM412.34,624.732c-117.3,0-212.391-95.092-212.391-212.392S295.04,199.949,412.34,199.949,624.732,295.04,624.732,412.34,529.64,624.732,412.34,624.732Z"/>
+  <symbol fill="currentColor" viewBox="0 0 24 24" id="search">
+    <path d="m21.415 19.665-5.416-5.416A7.496 7.496 0 1 0 14.25 16l5.416 5.416a.601.601 0 0 0 .848 0l.902-.902a.602.602 0 0 0 0-.848ZM9.896 14.994a5.097 5.097 0 1 1 0-10.195 5.097 5.097 0 0 1 0 10.195Z"/>
   </symbol>
-  <symbol fill="currentColor" viewBox="0 0 30 30" id="delete">
-    <path d="M24.536 20.298a1.499 1.499 0 010 2.12l-2.119 2.118a1.499 1.499 0 01-2.119 0L15 19.238l-5.298 5.298a1.499 1.499 0 01-2.12 0l-2.118-2.119a1.499 1.499 0 010-2.119L10.762 15 5.464 9.702a1.499 1.499 0 010-2.12l2.119-2.118a1.499 1.499 0 012.119 0L15 10.762l5.298-5.298a1.499 1.499 0 012.12 0l2.118 2.119a1.499 1.499 0 010 2.119L19.238 15z"/>
+  <symbol fill="currentColor" viewBox="0 0 24 24" id="delete">
+    <path d="M19.62 16.228a1.2 1.2 0 0 1 0 1.696l-1.696 1.696a1.2 1.2 0 0 1-1.696 0l-4.24-4.24-4.24 4.24a1.2 1.2 0 0 1-1.696 0l-1.696-1.696a1.2 1.2 0 0 1 0-1.696l4.24-4.24-4.24-4.24a1.2 1.2 0 0 1 0-1.696l1.696-1.696a1.2 1.2 0 0 1 1.696 0l4.24 4.24 4.24-4.24a1.2 1.2 0 0 1 1.696 0l1.696 1.696a1.2 1.2 0 0 1 0 1.696l-4.24 4.24 4.24 4.24Z"/>
   </symbol>
-  <symbol fill="currentColor" viewBox="0 0 30 30" id="settings">
```

Comment:
Settings shouldn't be deleted because used in `site/content/docs/5.3/utilities/colors.md`; see https://deploy-preview-1929--boosted.netlify.app/docs/5.3/utilities/colors/#oranges-colors